docs/source/configuration.rst

```diff
@@ -350,6 +350,74 @@ policy.
 
         *"It's some arcane magic from AT LEAST a decade ago."*
 
+Loop prevention
+---------------
+
+In order to prevent the bot from entering a loop (for example when there is
+another bot in the same channel, or if a user spams a command), it'll try to
+see if the next message to send is repeating too often in the last eight
+messages in the last few minutes. If that happens, the bot will send ``...``
+a few times before remaining silent::
+
+    <bot> I repeat myself!
+    <bot> I repeat myself!
+    <bot> I repeat myself!
+    <bot> I repeat myself!
+    <bot> I repeat myself!
+    # wanted to say: "I repeat myself"
+    <bot> ...
+    # wanted to say: "I repeat myself"
+    <bot> ...
+    # wanted to say: "I repeat myself"
+    <bot> ...
+    # silence, wanted to say: "..." instead of "I repeat myself"
+
+This doesn't affect non-repeating messages, and if enough time has passed
+between now and the last message sent, the loop prevention won't be triggered.
+
+This behavior can be configured with:
+
+* :attr:`~CoreSection.antiloop_threshold`: the number of repeating messages
+  before triggering the loop prevention.
+* :attr:`~CoreSection.antiloop_silent_after`: how many times the bot will send
+  the repeat text until it remains silent.
+* :attr:`~CoreSection.antiloop_window`: how much times (in seconds) since the
+  last message must pass before ignoring the loop prevention.
```

Comment:
```suggestion
* :attr:`~CoreSection.antiloop_window`: how much time (in seconds) since the
  last message must pass before ignoring the loop prevention.
```

First nemesi**S** in a long time. 😮